docs/release-notes/2.18.0.md

```diff
@@ -0,0 +1,93 @@
+* [PMM-6543](https://jira.percona.com/browse/PMM-6543): Users logged out of PMM after some period
+* [PMM-7910](https://jira.percona.com/browse/PMM-7910): MongoDB Query Collector never recovers from failed cursor (Thanks to Yann Rouillard for reporting this issue)
+* [PMM-6890](https://jira.percona.com/browse/PMM-6890): The "Environment" filter not working (Thanks to Daniel Guzman Burgos for reporting this issue)
+* [PMM-6451](https://jira.percona.com/browse/PMM-6451): Passing parameters between QAN and dashboards
+* [PMM-5368](https://jira.percona.com/browse/PMM-5368): "Failed to get PMM Server parameter" after config. (Thanks to Martin Wittwer for reporting this issue)
+* [PMM-5135](https://jira.percona.com/browse/PMM-5135): Query Example is often empty (Thanks to Mikhail Solovyev for reporting this issue)
+* [PMM-7673](https://jira.percona.com/browse/PMM-7673): IA: Actions column is transparent
+* [PMM-8123](https://jira.percona.com/browse/PMM-8123): Custom Queries Stop working after Upgrade PMM-Server to 2.18.0
+* [PMM-8083](https://jira.percona.com/browse/PMM-8083): PMM-Agent Config is failing
+* [PMM-7958](https://jira.percona.com/browse/PMM-7958): Databases cannot be deleted while Postgres is being monitored
+* [PMM-6553](https://jira.percona.com/browse/PMM-6553): Slow log size units are not defined in help
+* [PMM-6154](https://jira.percona.com/browse/PMM-6154): News panel does not work in fully isolated from internet environment
+* [PMM-5931](https://jira.percona.com/browse/PMM-5931): Graph and values are identical for TOTAL and N/A
+* [PMM-5538](https://jira.percona.com/browse/PMM-5538): Heavy Load with Distinct Queries on Slowlog enabled could cause no data being reported
+* [PMM-8095](https://jira.percona.com/browse/PMM-8095): Community footer link broken
+* [PMM-8088](https://jira.percona.com/browse/PMM-8088): [DBaaS] controller leaves kubeconfig behind in case of error
+* [PMM-7982](https://jira.percona.com/browse/PMM-7982): Query Analytics: Sorting element for the columns is hard to access
+* [PMM-7753](https://jira.percona.com/browse/PMM-7753): Edit DB Cluster shows wrong values by default
+* [PMM-7706](https://jira.percona.com/browse/PMM-7706): LDAP/AD users with Admin role mapping cannot see Integrated Alerting Option from left menu
+* [PMM-7315](https://jira.percona.com/browse/PMM-7315): Ensure DB cluster started to be deleted
+* [PMM-7184](https://jira.percona.com/browse/PMM-7184): DBaaS: Connection column showing different values after deleting DB cluster
+* [PMM-6676](https://jira.percona.com/browse/PMM-6676): Open Terms and Privacy in New tab
+* [PMM-6552](https://jira.percona.com/browse/PMM-6552): Failure to write config still registers
+* [PMM-5220](https://jira.percona.com/browse/PMM-5220): PMM Update for AMI instances shows no Log in Update Popup
+* [PMM-1880](https://jira.percona.com/browse/PMM-1880): Text message while query data is not yet loaded
```

Comment:
Looks like a PMM1 issue...not sure why it shows in pmm2.18.0 release notes...

Reply:
SO... should it be removed? Or is someone looking into it?